GovGen_Townhalls/Meeting4.md

```diff
@@ -25,7 +25,7 @@
 
 - Validator Onboarding Process & Timeline
   - Tuesday, 20th Feb- publish the Genesis - link will be up in a link in the README of https://github.com/atomone-hub/govgen-genesis
```

Comment:
```suggestion
  - Tuesday, 20th Feb- publish the Genesis - link is up in the [README.md](https://github.com/atomone-hub/govgen-genesis/blob/main/README.md#genesis-download-link) of https://github.com/atomone-hub/govgen-genesis
```

Reply:
Noticing also my horrible english here, it slipped through as we were rushing last minute changes